operators/rook/values-cluster.yaml

```diff
@@ -0,0 +1,187 @@
+# Defaults:
+# https://github.com/rook/rook/blob/release-1.15/deploy/charts/rook-ceph-cluster/values.yaml
+
+# TODO: adjust or remove this section when we have 3rd node
+cephClusterSpec:
+  mon:
+    # Set the number of mons to be started. Generally recommended to be 3.
+    # For highest availability, an odd number of mons should be specified.
+    count: 2
```

Comment:
If we leave this at 3, can we then make a kustomize patch for each deployment to adjust it since it's being customized?

Reply:
The intention is to test it on the new cluster, then keep it to the default (3) so not sure if we need it long term